eZ/Publish/Core/FieldType/Tests/ImageTest.php

```diff
@@ -811,4 +811,36 @@ public function provideInvalidDataForValidate()
             ],
         ];
     }
+
+    public function provideInputForValuesEqual(): array
+    {
+        return [
+            [
+                [
+                    'id' => $this->getImageInputPath(),
+                    'fileName' => 'Sindelfingen-Squirrels.jpg',
+                    'fileSize' => 23,
+                    'alternativeText' => 'This is so Sindelfingen!',
+                    'imageId' => '123-12345',
+                    'uri' => 'http://' . $this->getImageInputPath(),
+                    'width' => 123,
+                    'height' => 456,
+                ],
+                new ImageValue(
```

Comment:
CS, from the static analysis:

> Unhandled \eZ\Publish\Core\Base\Exceptions\InvalidArgumentType

Please report this exception in the method PHPDoc, but reference the API one - `\eZ\Publish\API\Repository\Exceptions\InvalidArgumentException`